File: add/codesnippet/CPP/a5b7accd-d3bc-4f0a-8140-_1.cpp
   // Create a SoapHexBinary object.
   array<Byte>^ bytes = gcnew array<Byte>(5){
      2,3,5,7,11
   };
   SoapHexBinary^ hexBinary = gcnew SoapHexBinary( bytes );
   Console::WriteLine( L"The SoapHexBinary object is {0}.", hexBinary );

File: add/codesnippet/CSharp/a5b7accd-d3bc-4f0a-8140-_1.cs
        // Create a SoapHexBinary object.
        byte[] bytes = new byte[]{ 2, 3, 5, 7, 11 };
        SoapHexBinary hexBinary = new SoapHexBinary(bytes);
        Console.WriteLine("The SoapHexBinary object is {0}.", 
            hexBinary.ToString());